### PR TITLE
Feature/braille transcript

### PR DESCRIPTION
**Pull request by jtreuting**
This contains a discrete amount of work that is needed for Braille Transcript but also for Embossing in general.

The following accommodations are added to the student package to make them available in TDS when selected for a student via ART:
- Braille Type
- Emboss Request Type

The Braille Transcript flag is also set in the student package and is calculated based on the student having Braille and Closed Captioning assigned.  It can not be assigned directly.

### CODE REVIEW

File: tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
  ******************************************************************************/
 package tds.dll.common.rtspackage.student.data;
 
+//import com.sun.org.apache.xpath.internal.operations.String;
```
Comment (millerts) [inline tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java]: Can this be removed?

File: tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java [hunk 2]
```diff
@@ -66,6 +71,10 @@
   protected String          streamlinedInterface;
   @XmlElement (name = "IllustrationGlossary")
   protected String          illustrationGlossary;
+  @XmlElement (name = "EmbossRequestType")
+  protected String          embossRequestType;
+  @XmlElement (name = "BrailleType")
+  protected String          brailleType;
```
Comment (millerts) [inline tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java]: I defer to you if it is worth changing but the properties below this one seem to be misaligned.  Or maybe it is just GitHub

File: tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java [hunk 3]
```diff
@@ -137,6 +159,48 @@ public void setClosedCaptioning (String value) {
   }
 
   /**
+   * Gets the value of the braillType property.
```
Comment (millerts) [inline tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java]: missing an `e` in braille here and below

Comment (jtreuting) [inline tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java]: Fixed

**Comment by millerts** [comment]
Approved

**Pull request merged**